helix-term/src/commands.rs

```diff
@@ -2745,9 +2843,13 @@ fn paste_clipboard_impl(editor: &mut Editor, action: Paste) {
         Ok(Some(transaction)) => {
             doc.apply(&transaction, view.id);
             doc.append_changes_to_history(view.id);
+            Ok(())
+        }
+        Ok(None) => Ok(()),
+        Err(e) => {
+            log::error!("Couldn't get system clipboard contents: {:?}", e);
+            bail!("Couldn't get system clipboard contents: {:?}", e);
```

Comment:
Clipboard is already using an `anyhow` error, ideally we should use `context` or similar instead.